### ISSUE TITLE
Barriers of entry to new guidelines

### ISSUE THREAD

**Issue by joeshearer** [w3c/silver#391]
Please consider a more streamlined approach to displaying the guidelines so that anyone not familiar with accessibility can start right away.

One of the things that always impressed be about WCAG 2.X is how easy it was for non-accessibility developers, testers and product owners to consume the criteria and just "get it." And not only get it, but to champion the guidelines to others on their teams.

In other words, you didn't really need to read all of the great information in the beginning to start doing accessibility.

In administering accessibility and VPAT training at my last company, this was crucial in building overall buy-in and quick wins, especially when there wasn't yet a formalized buy-in from leadership. The criteria themselves were quick chunks of currency and language small enough to communicate to one another. "This new functionality does not conform with criteria X, Y and Z in WCAG." Tester questions I dealt with usually had to do with the fuzziness of Section 508 criteria--not WCAG.

Even the way the criteria was laid out lent itself to easy scrolling to understand the criteria from its most basic high level. In 2.X, the criteria in the main view contains bold headings and a brief amount of information underneath. The supplemental **Understanding.**.. and **How to...** sections were easy to find and very predictably placed on the page as well, if you wanted that deeper dive. 

The way the guidelines and ratings are currently laid out and communicated in WCAG 3 makes me wonder if champions within organizations still soft on accessibility will be able to rally others and get them excited about accessibility.

Unlike WCAG 2.X, there seems to be more barriers of entry to understanding WCAG 3, such as:

- Understanding the ratings and conformance systems
- Effectively communicating the outcomes, which are listed like criteria, but have no specific numbers numbers associated with them (e.g., "7.4.1" is easier to communicate to different groups than "the first outcome associated with criterion 7.4").
- Getting used to lack of bolding that quickly, easily and definitively highlighted the criteria in 2.X
- Finding something solid to grasp onto and move forward with 
- Getting used to bigger scrolling gaps in between guidelines 

It's good to rethink the guidelines and make them more focused on the human experience, and I applaud this effort; I'm concerned, however, it could be very challenging to get the same traction as 2.X in smaller organizations, where accessibility champions are battling every day against all odds with competing interests.

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
